src/channel-qvm.lisp

```diff
@@ -0,0 +1,204 @@
+;;;; channel-qvm.lisp
+;;;;
+;;;; Authors: Sophia Ponte, Nikolas Tezak, Erik Davis
+
+(in-package #:qvm)
+
+;;; Implements a QVM that can interpret a rule-based noise model and
```

Comment:
This is to varying degrees documented elsewhere, but for the sake of orienting the reader I would suggest giving the reader a brief high-level picture of what is going on here (or someplace else, perhaps at the top of `noise-models.lisp`). It doesn't have to be much, but could include a brief summary of:
- what do you mean by a rule-based noise model? (mention the mechanism, that noise is applied either before or after a Quil instruction, based on syntactic properties of that instruction)
- how are models represented? no real details needed, can just refer to the NOISE-MODEL and NOISE-PREDICATE classes
- maybe a mention about the priority system (e.g. only one rule is applied in any context, the highest priority of the matching rules)